src/Pages/ItemList/Components/HeadPoster.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import styled from "styled-components";
+
+function HeadPoster({ category, series }) {
+  return (
+    <ItemDescription category={category}>
+      <div className="headTitle">{`${ITEM_DESC[category].name} ${
+        series ? (Number(series) > 0 ? series : "") : ""
```

Comment:
```suggestion
        series ? (+series > 0 ? series : "") : ""
```
이렇게 바꿔줄 수도 있습니다
```suggestion
        +series > 0 ? series : ""
```
이렇게도 가능할거 같은데 어떤가요?

Reply:
훨씬 좋은 방법 같고 동작도 잘 합니다 바로 적용했습니다!